### PR TITLE
Extend information in README

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | -
| License       | MIT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
-Chameleon System
-================
+<p align="center">
```
Comment (UlrichKu) [inline README.md]: This works (HTML)?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+    <a href="https://chameleonsystem.com" target="_blank">
+        <img src="https://demo.chameleon-system.de/chameleon/mediapool/8/ae/chameleon-logo_id3834.png">
```
Comment (UlrichKu) [inline README.md]: Is this URL stable in future?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+-------------
+
+The user manual and technical documentation in English language are currently in progress.
+
+The user manual in German language is available here: [https://manual.chameleon-system.de](https://manual.chameleon-system.de)
+
+Installation
+------------
+
+TBA
```
Comment (UlrichKu) [inline README.md]: Work in progress - also this PR?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+
+Chameleon System is licensed under the MIT License. See the complete license under `LICENSE` for details.
+Some Chameleon repositories also include works under other copyright notices; a `THIRD-PARTY-LICENSE` file will then
+provide further license information. 
+
+Reporting an Issue or a Feature Request
+---------------------------------------
+
+Please report issues and feature requests at the official [GitHub repository](https://github.com/chameleon-system/chameleon-system/issues).
+Issues for the basic Chameleon repositories chameleon-base and chameleon-shop are also centralized in this repository.
```
Comment (UlrichKu) [inline README.md]:
```suggestion
Issues for the Chameleon repositories chameleon-base and chameleon-shop are also handled centrally in this repository.
```

File: README.md (hunk 1 above)
Comment (umulmrum) [reply README.md]: Yes, see the rendered page on my fork.

File: README.md (hunk 2 above)
Comment (umulmrum) [reply README.md]: At least for now it's good enough - may change in the future.

File: README.md (hunk 3 above)
Comment (umulmrum) [reply README.md]: Nope, this needs yet some work by @proligde and me (I propose PRs only if they are ready to merge from my view, or add "WIP" to the title).

**Pull request merged**